api/lib/validation/schemaValidator.js

```diff
@@ -57,7 +57,7 @@ var getDaysInMonthInISOFormat = function() {
 };
 
 var getMetricTypes = function() {
-  var metricTypeEnum = ['memoryused'];
+  var metricTypeEnum = ['memoryused', 'memoryutil', 'responsetime', 'throughput']
```

Comment:
These are the metrics we can support with the events we get from loggregator. Once we support custom metrics, we will not have the metric type check anymore... So I would keep this way for now.